package.json

```diff
@@ -10,7 +10,7 @@
     "type": "git",
     "url": "https://github.com/RaspberryPiFoundation/Bits.git"
   },
-  "version": "1.2.0",
+  "version": "1.2.1",
```

Comment:
`npm i` needs to be run once this has been updated so the generated file also has the new version number. Probably unimportant, but feels like we should keep those numbers the same!